README.md

```diff
@@ -12,24 +12,25 @@ Run the `vpcanalyzer` CLI tool with one of the following commands.
 
 ### Global options
 ```
-      --dump-resources string    file path to store resources collected from the cloud provider
-  -o, --format string            output format; must be one of [json, txt, md, drawio, arch_drawio, svg, arch_svg, html, arch_html, debug]
-  -h, --help                     help for vpcanalyzer
-      --output-file string       file path to store results
-  -p, --provider string          collect resources from an account in this cloud provider
-  -q, --quiet                    runs quietly, reports only severe errors and results
-  -r, --region stringArray       cloud region from which to collect resources, can pass multiple regions
-      --resource-group string    resource group id or name from which to collect resources
-  -v, --verbose                  runs with more informative messages printed to log
-      --vpc string               CRN of the VPC to analyze
-  -c, --vpc-config stringArray   file paths to input configs, can pass multiple config files
+  -c, --config stringArray      file paths to input VPC configs, can pass multiple config files
+      --dump-resources string   file path to store resources collected from the cloud provider
+  -f, --filename string         file path to store results
+  -h, --help                    help for vpcanalyzer
+  -o, --output string           output format; must be one of [json, txt, md, drawio, arch_drawio, svg, arch_svg, html, arch_html, debug]
+  -p, --provider string         collect resources from an account in this cloud provider
+  -q, --quiet                   runs quietly, reports only severe errors and results
+  -r, --region stringArray      cloud region from which to collect resources, can pass multiple regions
+      --resource-group string   resource group id or name from which to collect resources
+  -v, --verbose                 runs with more informative messages printed to log
+      --version                 version for vpcanalyzer
+      --vpc string              CRN of the VPC to analyze
 ```
 
 ### Providing VPC configuration
-VPC configuration should be provided, using the `--vpc-config` option, as a `JSON` file produced by the [`cloud-resource-collector`](https://github.com/np-guard/cloud-resource-collector). Alternatively, VPC configuration can be read directly from a given account using the `--provider` flag.
+VPC configuration must be provided, using the `--config` option, as a `JSON` file produced by the [`cloud-resource-collector`](https://github.com/np-guard/cloud-resource-collector). Alternatively, VPC configuration can be read directly from a given account using the `--provider` flag.
```

Comment:
This is confusing to me -- either you *must* provide a configuration with `--config`, or there is an alternative, which is it?
Maybe you mean "A VPC configuration must be provided, in one of two ways. Either the `--provider` flag is used, in which case the VPC configuration will be extracted directly from a given account, or a configuration object is provided by the user using the `--config` option. A configuration object can be independently produced by the [`cloud-resource-collector`](https://github.com/np-guard/cloud-resource-collector)"

Reply:
right, updated.